lieer/gmailieer.py

```diff
@@ -785,6 +782,9 @@ def set (self, args):
     if args.remove_local_messages:
       self.local.config.set_remove_local_messages (True)
 
+    if args.limit is not None:
```

Comment:
How do you unset the limit?

Reply:
I have just added the way to unset it by using `gmi set --limit 0`.